cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/WorkflowHttpHandler.java

```diff
@@ -323,4 +341,85 @@ private WorkflowToken getWorkflowToken(String namespaceId, String appName, Strin
     }
     return store.getWorkflowToken(workflowId, runId);
   }
+
+  @GET
+  @Path("/apps/{app-id}/workflows/{workflow-id}/runs/{run-id}/localdatasets")
```

Comment:
Do you think it also makes sense to be able to retrieve an individual local dataset? With its full spec, exactly like we do for GET .../data/datasets/{name}